.github/workflows/build-and-test.yaml

```diff
@@ -40,5 +40,5 @@ jobs:
         uses: codecov/codecov-action@v2
         with:
           files: lcov/total_coverage.info,coveragepy/.coverage
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
Do we also have to change this?

Reply:
This is a temporary change. I am currently investigating CodeCov related errors to resolve them. Once resolved, this fix will not be necessary.

In another discussion, this error was tentatively tolerated, so the change will be reverted.

Reply:
d3ab689